addons/project_todo/static/src/views/todo_form/todo_form_controller.js

```diff
@@ -0,0 +1,57 @@
+/** @odoo-module **/
+
+import { FormController } from "@web/views/form/form_controller";
+import { TodoEditableBreadcrumbName } from "@project_todo/components/todo_editable_breadcrumb_name/todo_editable_breadcrumb_name";
+import { TodoDoneCheckmark } from "@project_todo/components/todo_done_checkmark/todo_done_checkmark";
+
+import { onWillStart } from "@odoo/owl";
+
+/**
+ *  The FormController is overridden to be able to manage the edition of the name of a to-do directly
+ *  in the breadcrumb as well as the mark as done button next to it.
+ */
+
+export class TodoFormController extends FormController {
+    setup() {
+        super.setup();
+        onWillStart(async () => {
+            this.projectAccess = await this.model.orm.call(
+                'project.task',
+                'get_project_access_right',
+                [],
+            );
```

Comment:
We can use `hasGroup` method defined in user service [1] instead of calling a custom method. 🙂 
The user service is already stored in the parent class in `user` attribute [2] and so we could do something like this:
```suggestion
            this.projectAccess = await this.user.hasGroup("project.group_project_user")
```

[1] https://github.com/odoo/odoo/blob/a5036864b11890c197d636c461a42720fbce6227/addons/web/static/src/core/user_service.js#L46-L48

[2] https://github.com/odoo/odoo/blob/5de54e14c4d4496abde07a94cab770e2e811c38b/addons/web/static/src/views/form/form_controller.js#L101